pkg/cloudprovider/kubevirt/zones.go

```diff
@@ -60,10 +60,14 @@ func (z *zones) getZoneByInstanceID(ctx context.Context, instanceID string) (clo
 
 	// Take over failure domain and region from node where the instance is running on.
 	zone := cloudprovider.Zone{}
-	if failureDomain, ok := node.ObjectMeta.Labels[corev1.LabelZoneFailureDomain]; ok {
+	if failureDomain, ok := node.ObjectMeta.Labels["topology.kubernetes.io/zone"]; ok {
```

Comment:
Please add a TODO comment here to update the string to `corev1.LabelZoneFailureDomainStable` as soon as the k8s.io/api/core/v1 dependency is updated to 1.17 or greater.

Reply:
Done